docs/index.md

```diff
@@ -0,0 +1,320 @@
+# lolex
+
+> JavaScript implementation of the timer APIs
+
+**Implements:**
+
+* [`setTimeout`](#settimeoutcallback-timeout-args)
+* [`clearTimeout`](#cleartimeoutid)
+* [`setImmediate`](#setimmediatecallback-args)
+* [`clearImmediate`](#clearimmediateid)
+* [`setInterval`](#setintervalcallback-timeout-args)
+* [`clearInterval`](#clearintervalid)
+* [`requestAnimationFrame`](#requestanimationframecallback)
+* [`cancelAnimationFrame`](#cancelanimationframeid)
+* and many more ...
```

Comment:
Can we please write those down explicitly? Those are `Date`, `nextTick`, `queueMicrotask`, and `performance.now` IIRC.